api/notd/twitter_manager.py

```diff
@@ -3,27 +3,35 @@
 import os
 import urllib.parse as urlparse
 import uuid
-from typing import Optional
+from typing import List, Optional
 
 from core.exceptions import FoundRedirectException
 from core.exceptions import NotFoundException
 from core.http.basic_authentication import BasicAuthentication
+from core.queues.sqs_message_queue import SqsMessageQueue
 from core.requester import Requester
 from core.util import date_util
 from core.util import dict_util
+from core.util import list_util
+from notd.store.schema import TwitterProfilesTable
+from notd.model import RetrievedTwitterProfile
 
+from notd.messages import UpdateAllTwitterUsersMessageContent
 from notd.model import Signature
 from notd.model import TwitterCredential
 from notd.store.retriever import Retriever
 from notd.store.saver import Saver
 
+_TWITTER_BEARER_TOKEN=os.environ("TWITTER_BEARER_TOKEN")
```

Comment:
dont read from environ outside of the entry pojnts (application.py, worker.py, scripts). this should be read at an entry point and passed into the constructor